e2etests/empty.go

```diff
@@ -0,0 +1,3 @@
+package e2etests
+
+// Empty file with no build tag to keep the Go compiler happy.
```

Comment:
Not a PR comment but why does go require an empty file for compilation?

Reply:
Basically, some go commands require that there is at least one file in each package. Since the other file in this package has a build tag (that's the `+build e2e` line on top), it is not included by default unless the go command passes `-tags e2e`, and so go complains the package is empty.

(Build tags are just a useful way to selectively include files under certain conditions (for example, if you need to have a different implementation depending on the OS, you can do that). In this case, I gave all e2etests a build tag of `e2e` so that they don't get run if someone simply runs `go test ./...` to run unit tests, they get run only if someone explicitly asks for e2es.

Ref: https://golang.org/pkg/go/build/#hdr-Build_Constraints)

Reply:
gotcha. thanks!